docs.go

```diff
@@ -196,3 +277,235 @@ func prepareUsage(command *Command, usageText string) string {
 
 	return usage
 }
+
+type (
+	cliTabularAppTemplate struct {
+		AppPath          string
+		Name             string
+		Usage, ArgsUsage string
```

Comment:
Nitpick: put ArgsUsage in next line

Reply:
Done in d8caf8b6054565043aa06fffe534d8e522e7ced2